web/server/vue-cli/src/components/Layout/PersonalAccessTokenBtn.vue

```diff
@@ -0,0 +1,182 @@
+<template>
+  <div>
+    <confirm-dialog
+      v-model="deleteDialog"
+      confirm-btn-color="error"
+      confirm-btn-label="Delete access token"
+      cancel-btn-color="primary"
+      @confirm="deleteToken(token.name)"
+    >
+      <template v-slot:title>
+        Delete Personal Access Token
+      </template>
+
+      <template v-slot:content>
+        <p>
+          You have selected to delete token with name
+          <strong>{{ token.name }}</strong>.
+        </p>
+      </template>
+    </confirm-dialog>
+
+    <confirm-dialog
+      v-model="newDialog"
+      confirm-btn-label="OK"
+      @confirm="newDialog = false"
+    >
+      <template v-slot:title>
+        New Personal Access Token
+      </template>
+
+      <template v-slot:content>
+        <p>
+          Token: <strong>{{ token.token }}</strong>
+        </p>
+        <p>
+          Name: <strong>{{ token.name }}</strong>
+        </p>
+        <p>
+          Expiration: <strong>{{ token.expiration }}</strong>
+        </p>
+        <p>
+          Description: <strong>{{ token.description }}</strong>
+        </p>
+        <p>
+          Make sure to save this token, because it won't be available anymore
+          after closing this window.
+        </p>
+      </template>
+    </confirm-dialog>
```

Comment:
It would be nice if instead of a Cancel button here, there was a Copy to clipboard button, or something similar.